specs/protocol/isthmus/l1-attributes.md

```diff
@@ -0,0 +1,37 @@
+# L1 Block Attributes
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+The L1 block attributes transaction is updated to include the operator fee parameters.
+
+| Input arg         | Type    | Calldata bytes | Segment |
+| ----------------- | ------- | -------------- | ------- |
+| {0x098999be}      |         | 0-3            | n/a     |
```

Comment:
what is this?

Reply:
That's the call signature for the function. Those 4 bytes are the first four bytes of `keccak("setL1BlockValuesIsthmus()")`. I'll clarify this in the spec.

Reply:
Bump on adding this to the spec to make it clear what this is